.drone.yml

```diff
@@ -374,7 +374,17 @@ pipeline:
     when:
       branch: [dev]
       event: [push]
-
+  
+  build_solidity_docs:
+    group: build_artifacts
+    image: kowalatech/solidoc
+    commands:
+      - make dev_docker_images
```

Comment:
Do you really need all docker images? maybe just `make dev_solidocs_docker_image` no? Also I think we can push this image to docker hub so we don't have to build it every time

Reply:
That should definitely not be there!

Reply:
My mistake using copy-paste!